src/utils/isTemplateProp/isTemplateProp.spec.js

```diff
@@ -0,0 +1,10 @@
+import { expect } from 'chai'
```

Comment:
In my opinion this test is too specific. If we cover the underlying functionality which delivers this results, we do not need to have specific unit tests for each use case.

I believe it's enough to write a unit test for `pick` util. If it passes, everything else using it under the hood passes too.